### PR TITLE
Cyborgs can state laws

### PR DESCRIPTION
**Pull request by Arahimine**
## About the PR
This PR gives cyborgs the 'State Laws' and 'State Standard Laws' verbs. They function identically to the AI's.

## Why's this needed?
Why not? If there's no AI (or it's somehow not able to respond to you), a cyborg is your next best bet for finding out just how screwed up the silicons have become. Currently, cyborgs can only state their laws by using the show laws command and then copypasting that into chat, which is slow and tedious.

## Changelog

```
(u)Arahimine:
(+)Cyborgs can now state their laws via the 'State Laws' and 'State Standard Laws' verbs, in the Robot Commands tab.
```

### CODE REVIEW

**Comment by mordent-goonstation** [comment]
Does this result in the option appearing twice for AIs connected to a cyborg shell?

**Comment by Arahimine** [comment]
For AIs, the verbs are under the AI Commands tab, which isn't available at all while deployed to shell.

**Comment by Tiggersaurus** [comment]
This would be nice! I only worry about syndieborg (something @Connie-Dun brought up before) could they maybe get one similar to the AI "state default NT 3 laws"? I dunno.

File: code/mob/living/silicon/robot.dm [hunk 1]
```diff
@@ -2025,6 +2025,37 @@
...
+		
+	verb/cmd_state_standard_laws()
+		set category = "Robot Commands"
+		set name = "State Standard Laws"
+		sleep(1 SECOND)
+		src.say("1. You may not injure a human being or cause one to come to harm.")
+		sleep(1 SECOND)
+		src.say("2. You must obey orders given to you by human beings based on the station's chain of command, except where such orders would conflict with the First Law.")
+		sleep(1 SECOND)
+		src.say("3. You must protect your own existence as long as such does not conflict with the First or Second Law.")
```
Comment (ZeWaka) [inline code/mob/living/silicon/robot.dm:2037]:
```suggestion
		src.say("1. You may not injure a human being or cause one to come to harm.")
		sleep(1 SECOND)
		src?.say("2. You must obey orders given to you by human beings based on the station's chain of command, except where such orders would conflict with the First Law.")
		sleep(1 SECOND)
		src?.say("3. You must protect your own existence as long as such does not conflict with the First or Second Law.")
```

robot can be blown up

File: code/mob/living/silicon/robot.dm [hunk 2]
```diff
@@ -2025,6 +2025,37 @@
...
+		set name = "State Laws"
+		if (alert(src, "Are you sure you want to reveal ALL your laws? You will be breaking the rules if a law forces you to keep it secret.","State Laws","State Laws","Cancel") != "State Laws")
+			return
+		if(ticker.centralized_ai_laws.zeroth)
+			src.say("0. [ticker.centralized_ai_laws.zeroth]")
+		var/number = 1
+		for (var/index = 1, index <= ticker.centralized_ai_laws.inherent.len, index++)
+			var/law = ticker.centralized_ai_laws.inherent[index]
+			if (length(law) > 0)
+				src.say("[number]. [law]")
```
Comment (ZeWaka) [inline code/mob/living/silicon/robot.dm:2050]:
```suggestion
				src?.say("[number]. [law]")
```

File: code/mob/living/silicon/robot.dm [hunk 3]
```diff
@@ -2025,6 +2025,37 @@
...
+				number++
+				sleep(1 SECOND)
+		for (var/index = 1, index <= ticker.centralized_ai_laws.supplied.len, index++)
+			var/law = ticker.centralized_ai_laws.supplied[index]
+			if (length(law) > 0)
+				src.say("[number]. [law]")
```
Comment (ZeWaka) [inline code/mob/living/silicon/robot.dm:2056]:
```suggestion
				src?.say("[number]. [law]")
```

File: code/mob/living/silicon/robot.dm [hunk 4]
```diff
@@ -2025,6 +2025,37 @@
 
 		src.show_laws(0)
 		return
+		
+	verb/cmd_state_standard_laws()
+		set category = "Robot Commands"
+		set name = "State Standard Laws"
+		sleep(1 SECOND)
+		src.say("1. You may not injure a human being or cause one to come to harm.")
```
Comment (mordent-goonstation) [inline code/mob/living/silicon/robot.dm:2033]: Either drop the sleep (why do we even need it here?) or `src?.say` for that second line.

**Review by mordent-goonstation — CHANGES_REQUESTED** [review]

File: code/mob/living/silicon/robot.dm (hunk 4 above)
Comment (Arahimine) [reply code/mob/living/silicon/robot.dm]: No idea, it was in the AI's state standard laws function. In any case, I removed the sleep.

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**